### DISCUSSION TITLE
Google Assistant via android auto

### DISCUSSION THREAD

**Discussion by thetangstr** [tesla-android/issue-tracker#231] [General]
First of all, 
This is awesome! Thank you so much for making this free and open source. 

I didn't realize about how much I rely on Google Maps and Assistant after I got my Model3. 

Is google assistant supposed to work via AA? I can click on the icon but almost seems like it couldn't pick up anything from the Mic. I had changed Audio setting to bluetooth in Advanced Setting as suggested in the install guide. I tried with two different carlinkits (one wired, one wireless), same behavior.  Anything else I should do?

**Comment by brswattt** [comment]
> First of all, This is awesome! Thank you so much for making this free and open source.
> 
> I didn't realize about how much I rely on Google Maps and Assistant after I got my Model3.
> 
> Is google assistant supposed to work via AA? I can click on the icon but almost seems like it couldn't pick up anything from the Mic. I had changed Audio setting to bluetooth in Advanced Setting as suggested in the install guide. I tried with two different carlinkits (one wired, one wireless), same behavior. Anything else I should do?

Did you ever figure this out? I haven't been able to get it working.
